src/components/Grid/Cell.tsx

```diff
@@ -60,11 +56,27 @@ export const Cell = memo(
     const selectionBorderLeft = rightOfSelectionBorder || rightOfFocus || isFocused;
     const selectionBorderTop = belowSelectionBorder || belowFocus || isFocused;
 
+    const cellRef = useRef<HTMLDivElement>(null);
+
+    useEffect(() => {
+      if (!rowAutoHeight) return;
+      else if (cellRef.current) {
```

Comment:
This `else` is unnecessary since the previous `if` statement returns. Making this an `if/else` communicates that these are dependent conditions, but they appear to be independent to me: 

If row auto height isn't defined, return out of the function. (Independently of that) if cell ref has a value, then set the height based on that.